server/models/index.ts

```diff
@@ -0,0 +1,5 @@
+import User from './User.model';
+
+export {
+  User,
+};
```

Comment:
A single entry point for all models.  All models should be exported from here.  This way, instead of doing something like the following:

```js
import ModelA from 'server/models/ModelA.model.ts';
import ModelB from 'server/models/ModelB.model.ts';
import ModelC from 'server/models/ModelC.model.ts';
```

It would just be:

```js
import {
  ModelA,
  ModelB,
  ModelC,
} from 'server/models';
```